ext/liquid_c/block.c

```diff
@@ -334,7 +332,7 @@ static VALUE block_body_nodelist(VALUE self)
                 break;
             }
 
-            case OP_POP_WRITE_VARIABLE:
+            case OP_RENDER_VARIABLE_RESCUE:
```

Comment:
We just needed a marker for adding a variable placeholder to the nodelist.  I decided to make `OP_POP_WRITE` generic (rather than variable specific) before exposing it to ruby.